Add README.md files to improve documentation UX (#4556)
vmware-tanzu/kubeapps: docs/howto/README.md

# How-to guides

How-to guides can be thought of as recipes, directions that guide the reader through the steps to achieve a specific end. They’ll help you achieve an end result but may require you to understand and adapt the steps to fit your specific requirements. Here you’ll find short answers to “How do I….?” types of questions.

| How-to-guides                                                                          | Get stuff done                                                                                                                            |
| -------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
| [Access Control](./access-control.md)                                                  | Establish how users will authenticate with the Kubernetes clusters on which Kubeapps operates.                                            |

> | [Basic Form Support](./basic-form-support.md)                                          | Configure your Helm chart in order to present this basic form to the users.                                                               |

Reviewer: ```suggestion
| [Basic Form Support](./basic-form-support.md)                                          | Configure your Helm chart in order to present a simple intuitive form during installation.                                                               |
```
